Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -526,6 +526,29 @@ InitializeSmbiosInfo (
   //
   // SMBIOS_TYPE_PROCESSOR_INFORMATION : TBD
   //
+  //
+  // SMBIOS_TYPE_PHYSICAL_MEMORY_ARRAY
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_PHYSICAL_MEMORY_ARRAY,
+    1, "2");
+
+  //
+  // SMBIOS_TYPE_MEMORY_DEVICE
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    1, "Controller0-ChannelA");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    2, "BANK 0");
```

Comment:
EHL can have up to 4 DIMMs. If multiple DIMMs are used, there will have multiple type17 entries.

The device locator and bank locator for each are:

Memory down:
```
Controller0-ChannelA, BANK0
Controller0-ChannelA, BANK0
Controller0-ChannelB, BANK1
Controller0-ChannelB, BANK1

```
socketed:
```
Controller0-ChannelA-DIMM0, BANK0
Controller0-ChannelA-DIMM1, BANK0
Controller0-ChannelB-DIMM0, BANK1
Controller0-ChannelB-DIMM1, BANK1

```